src/routes/events/[query].json/+server.ts

```diff
@@ -0,0 +1,24 @@
+import type { RequestEvent } from '@sveltejs/kit';
+import type { RouteParams } from './$types';
+import { DEBUG_FLAG_ENABLED } from '$env/static/private';
```

Comment:
Import DEBUG_FLAG_ENABLED from server flags.
```suggestion
import { DEBUG_FLAG_ENABLED } from '$lib/server/flags';
```

Reply:
Followed up by #581.

Reply:
Resolved by #589.